File: #Task-03/Js/Sub-Task 3/Sub-Task 3.js
var n=process.argv.slice(2);
if ((n%2)==1){
    x=Math.floor(n/2);
    var c=n-x-1;
    var m=1;
    var s1=-1;
    var s2=1;
    for(var g=0;g<n;g++){
        if (c!=0){
            process.stdout.write(" ".repeat(c));
        }
        console.log("*".repeat(m))
        m+=(s2*2)
        c+=(s1*1)
        if (c==0){
            s1=1}
        if (m>n){
            s2=-1
            m=n-2}
    }

}else{
    console.log("Pls Enter A Even Number");
}

File: #Task-03/Python/Sub-Task 3/Sub-Task3.py
while True:
    n=int(input())
    if n%2!=0:
        c=n-(n//2)-1
        print(c)
        m=1
        s1=-1
        s2=1
        for g in range(n):
            if c!=0:
                print(" "*c,end="")
            print("*"*m)
            m+=(s2*2)
            c+=s1
            if c==0:
                s1=1
            if m>n:
                s2=-1
                m=n-2
    else:
        print("Enter A Odd Number")

